docs/src/manual/construction.md

```diff
@@ -14,31 +12,52 @@ interval(Inf) # not valid since infinity is not part of an interval
 interval(3, 2) # not valid since the lower bound is strictly greater than the upper bound
 ```
 
-The submodule `IntervalArithmetic.Symbols` exports the infix operator `..` as an alias for `interval`.
+The above intervals came with a decoration. Decorations are flags, or labels, attached to intervals to indicate the status of a given interval. Decorated intervals provide valuable information on the result of evaluating a function on an initial interval.
+
+Suppose that a decorated interval $(X, d)$ is the result of evaluating a function $f$, or the composition of a sequence of functions, on an initial decorated interval $(X_0, d_0)$. The meaning of the resulting decoration $d$ is as follows:
+- `com` (common): $X$ is a closed, bounded, non-empty subset of the domain of $f$; $f$ is continuous on the interval $X$; and the resulting interval $f(X)$ is bounded.
+- `dac` (defined and continuous): $X$ is a non-empty subset of $\mathrm{Dom}(f)$, and $f$ is continuous on $X$.
+- `def` (defined): $X$ is a non-empty subset of $\mathrm{Dom}(f)$, i.e. $f$ is defined at each point of $X$.
+- `trv` (trivial): always true; gives no information
+- `ill` (ill-formed): NaI (Not an Interval) (an error occurred), e.g. $\mathrm{Dom}(f) = \emptyset$.
+Decorations follow the ordering `com > dac > def > trv > ill`.
+
+When constructing intervals, if no decoration is explicitly specified, then it is initialised with a decoration according to the underlying bare interval:
+- `com`: if the interval is non-empty and bounded
+- `dac`: if the interval is unbounded
+- `trv`: if the interval is empty
+- `ill`: if the interval is a NaI
```

Comment:
Isn't `def` missing here? (I think the subtlety wrt `dac` is that continuity is not guaranted)

Reply:
When constructing intervals, I believe we cannot have `def`. This decoration can only be produced by some function.